diff --git a/internal/check/valid_owner.go b/internal/check/valid_owner.go
index a264bcc..8600c57 100644
--- a/internal/check/valid_owner.go
+++ b/internal/check/valid_owner.go
@@ -16,7 +16,6 @@ import (
 const scopeHeader = "X-OAuth-Scopes"
 
 var reqScopes = map[github.Scope]struct{}{
-	github.ScopeReadOrg: {},
 }
 
 type ValidOwnerConfig struct {
diff --git a/pkg/codeowners/owners.go b/pkg/codeowners/owners.go
index 6910bd2..2ff8e2f 100644
--- a/pkg/codeowners/owners.go
+++ b/pkg/codeowners/owners.go
@@ -51,7 +51,7 @@ func openCodeownersFile(dir string) (io.Reader, error) {
 			continue
 		}
 
-		f := path.Join(pth, "CODEOWNERS")
+		f := path.Join(pth, "OWNERS")
 		_, err = fs.Stat(f)
 		switch {
 		case err == nil:
